controllers/cloudstackmachine_controller.go

```diff
@@ -179,6 +186,15 @@ func (r *CloudStackMachineReconciliationRunner) GetOrCreateVMInstance() (retRes
 	return ctrl.Result{}, err
 }
 
+func processUserData(data []byte, r *CloudStackMachineReconciliationRunner) string {
```

Comment:
Per the comment, can you name this func processCustomMetadata()?

Reply:
updated.